examples/src/lib.rs

```diff
@@ -119,6 +143,12 @@ pub enum ExampleType {
         #[structopt(short = "n", default_value = "1048576")]
         sequence_length: usize,
     },
+    /// Compute a Fibonacci sequence using trace table with 2 registers in `f64` field.
+    FibSmall {
+        /// Length of Fibonacci sequence; must be a power of two
+        #[structopt(short = "n", default_value = "67072")]
```

Comment:
That is not a power of two (and hence the default case fails). The closest one is the one below
```suggestion
        #[structopt(short = "n", default_value = "65536")]
```

Reply:
Thanks, I've updated it in the latest commit. Please see [80c03a8](https://github.com/novifinancial/winterfell/pull/111/commits/80c03a80aaf7359be794e12aeb93f2d4256b900d). I checked the default case in my local system, and I was able to generate the proof of the default case.

Reply:
Great!